### PR TITLE
docs(tooltips): Migrate tooltips to API Docs

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-205

- [x] Dev LGTM
- [x] Dev LGTM

Migrate `tooltips` to API Documentation.

### CODE REVIEW

File: src/tooltips/tooltips.js [hunk 1]
```diff
@@ -4,6 +4,31 @@
...
+ * <rx-button tooltip="...">
+ * </pre>
+ * 
+ * If you're creating your own custom directive, it's fine to use the `tooltip` 
+ * directive inside of your directive's template.  See the tooltips component 
+ * {@link /#/components/tooltips demo} for example usage.
+ *
+ *
+ * 
+ * The [Angular-UI Bootstrap Tooltip](https://github.com/angular-ui/bootstrap/tree/master/src/tabs)
```
Comment (100stacks) [inline src/tooltips/tooltips.js]: @CITguy @bry4n `https://github.com/angular-ui/bootstrap/tree/master/src/tabs` is referenced for the `tooltips` component.
Should it point to:
`https://github.com/angular-ui/bootstrap/tree/master/src/tooltip` instead?

**Comment by bry4n** [comment]
@100stacks @CITguy yes it should point to tooltip

File: src/tooltips/tooltips.js (hunk 1 above)
Comment (CITguy) [inline src/tooltips/tooltips.js]: Yes, please update.

**Comment by 100stacks** [comment]
@bry4n @CITguy  okay thanks.  I'll update the reference.

**Comment by CITguy** [comment]
@100stacks, please rebase before you update to correct Travis error.

**Comment by 100stacks** [comment]
@CITguy, okay will do :+1:

**Comment by bry4n** [comment]
LGTM

File: src/tooltips/README.md [hunk 2]
```diff
@@ -1,11 +1,3 @@
...
-This component provides styles and a demo for the [the Angular-UI Bootstrap Tooltip plugin](https://github.com/angular-ui/bootstrap/tree/master/src/tabs), which is included as a dependency for EncoreUI.
-
-Note that the `tooltip` directive and its optional attributes can *only* be applied to raw HTML elements. They can't be applied to directives like this: `<rx-button tooltip="...">`
-
-If you're creating your own custom directive, it's fine to use the `tooltip` directive inside of your directive's template.
-
-## Usage
-
-Usage is the exact same as demoed on the Angular-UI Bootstrap site. See [the Angular-UI Bootstrap Docs](http://angular-ui.github.io/bootstrap/) for further guidance on usage and configuration of this component.
+The tooltips component and its optional attributes provide styles to raw HTML elements and custom directive templates.
```
Comment (CITguy) [inline src/tooltips/README.md]: Please remove "and its optional attributes". This references the directive, not the component.

File: src/tooltips/tooltips.js [hunk 3]
```diff
@@ -4,6 +4,31 @@
  * @description
  * # tooltips Component
  *
- * [TBD]
+ * The tooltip component and its optional attributes provide styles to raw HTML
```
Comment (CITguy) [inline src/tooltips/tooltips.js]: Same as README

File: src/tooltips/tooltips.js [hunk 4]
```diff
@@ -4,6 +4,31 @@
...
+ * // DOES NOT WORK 
+ * 
+ * <rx-button tooltip="...">
+ * </pre>
+ * 
+ * If you're creating your own custom directive, it's fine to use the `tooltip` 
+ * directive inside of your directive's template.  See the tooltips component 
+ * {@link /#/components/tooltips demo} for example usage.
+ *
+ *
```
Comment (CITguy) [inline src/tooltips/tooltips.js]: Please reduce to one blank line between paragraphs.

**Comment by 100stacks** [comment]
@CITguy okay comments incorporated. :+1:

**Comment by CITguy** [comment]
LGTM

**Pull request merged**